src/main/java/ome/services/sessions/SessionBean.java

```diff
@@ -90,13 +90,14 @@ public Session createUserSession(final long timeToLiveMs,
             currentSession = null;
         }
 
+        final String agent = cd.getContext().get("omero.agent");
```

Comment:
A similar check that the one added line 184 should probably be added there to i.e.
``if (cd.size() > 0)``

Reply:
Technically I don't think that's a possible condition @jburel. I only found the other case in an obscure mock'd test case. However it can't hurt to have.